vue2-vuetify/src/layouts/ArrayLayoutRenderer.vue

```diff
@@ -1,24 +1,24 @@
 <template>
-  <v-card v-if="control.visible" :class="styles.arrayList.root" elevation="0">
```

Comment:
The elevation can be controlled by passing in a custom class, if required.

Reply:
Yes, it can be passed as either a prop or a css class like `elevation-0` https://vuetifyjs.com/en/styles/elevation/